cluster/gce/gci/configure.sh

```diff
@@ -387,7 +387,7 @@ function load-docker-images {
 # and places them into suitable directories. Files are placed in /home/kubernetes.
 function install-kube-binary-config {
   cd "${KUBE_HOME}"
-  local -r server_binary_tar_urls=( $(split-commas "${SERVER_BINARY_TAR_URL}") )
+  local -r server_binary_tar_urls=( (split-commas "${SERVER_BINARY_TAR_URL}") )
```

Comment:
why is this change being made?

Reply:
some of these changes are not immediately obvious, it would be helpful to explain why we are changing these other than "shellcheck passes now". passing != correct shell, it's not perfect.